src/pages/Admin/Charity/EditProfile/HideBGTipCheckbox.tsx

```diff
@@ -7,7 +7,7 @@ export default function HideBGTipCheckbox() {
       <CheckField<FV> name="hide_bg_tip" classes="text-sm">
         Opt out of tip-based donations
       </CheckField>
-      <span className="text-xs sm:text-sm italic text-navy-l1">
+      <span className="text-xs sm:text-sm text-navy-l1">
```

Comment:
```suggestion
      <span className="text-xs sm:text-sm italic text-navy-l1">
```
slug description contains bold words which won't show under italic - so slug desscription is not italic. Made tip help text also not italic for uniformity

Reply:
I've been staring at the new slug help text for a bit and I _think_ that we should prob:
- [ ] drop the bold text on slug helptext/description
- [ ] revert to using italic

It just seems too similar to the labels if we remove the italic. Wdyt?

Reply:
> It just seems too similar to the labels if we remove the italic. Wdyt?

To me smaller and lighter helper text is enough to distinguish it from the labels. Nonetheless I've applied to changes.

<img width="1015" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/b6d2e574-1ee2-46a4-bf53-815ba1e5c79c">